scripts/commands/zone.lua

```diff
@@ -15,264 +15,264 @@ cmdprops =
 ---------------------------------------------------------------------------------------------------
 local zone_list =
 {
-    { 0x14, 0xA9, 1 }, -- Phanauet Channel
-    { 0x14, 0xAA, 2 }, -- Carpenters' Landing
-    { 0x14, 0x84, 3 }, -- Manaclipper
```

Comment:
I think these are literal byte values for auto translate tables? I can see an argument being made to keep hex.